testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/AbstractKeycloakTest.java

```diff
@@ -644,16 +644,19 @@ public void setTimeOfDay(int hour, int minute, int second, int addSeconds) {
 
     /**
      * Sets time offset in seconds that will be added to Time.currentTime() and Time.currentTimeMillis() both for client and server.
+     * Moves time on the remote Infinispan server as well if the Infinispan is used.
      *
      * @param offset
      */
     public void setTimeOffset(int offset) {
+        testingClient.testing().setInfinispanTimeTask(offset);
```

Comment:
Can we make it part of one server call inside `invokeTimeOffset`?

Reply:
done